Jenkinsfile

```diff
@@ -1,11 +1,46 @@
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '10')),
+    parameters([
+        booleanParam(defaultValue: false,
+            description: 'Override distfeeds.conf links',
+            name: 'OVERRIDE_PACKAGES'),
+        stringParam(defaultValue: '',
+            description: 'Works only with OVERRIDE_PACKAGES set to true.\
+                          Path to a directory containing the packages folder,\
+                          if null defaults to ${JOB_PATH}/artifact/bin/pistachio',
+            name: "PACKAGES_PATH"),
+        stringParam(defaultValue: '',
+            description: 'Path to the job that needs to be tested, if null downloads newest stable release',
+            name: "JOB_PATH"),
+    ])
+])
+
 node('boardfarm') {
     stage("Configure") {
         deleteDir()
 
+        def image_path = "http://downloads.creatordev.io/openwrt/ci40-v1.1.0/pistachio/marduk"
+        def image_name = "openwrt-v1.1.0-pistachio-marduk-squashfs-factory.ubi"
```

Comment:
IMO, rather than hard coding, we should allow this to be overriden from CI to test other images by using Build With Parameters.  For the best example, due to the current known issue with our CI CreatorDev/openwrt#217, we would need to re-build the images using ImageBuilder and host them on a different web server.  It would then be necessary to run the BoardFarm tests against such custom image and hence the path/url to it would need to be supplied as a parameter to simply run the tests on it